drivers/at86rf2xx/at86rf2xx_rf_power.c

```diff
@@ -0,0 +1,108 @@
+                                              0x10, 0x0f, 0x0e, 0x0d, 0x0c, 0x0b,
+                                              0x09, 0x91, 0x08, 0x07, 0x05, 0x27,
+                                              0x04, 0x03, 0x02, 0x01, 0x00, 0x86,
+                                              0x40, 0x84, 0x83, 0x82, 0x80, 0xc1,
+                                              0xc0 };
+const int16_t _212b_rx_sens_to_dbm[16] = { -110, -98, -94, -91, -88, -85, -82,
+                                           -79, -76, -73, -70, -67, -63, -60, -57,
+                                           -54 };
+const uint8_t _212b_dbm_to_rx_sens[57] = { 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+                                           0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+                                           0x01, 0x01, 0x01, 0x01, 0x02, 0x02,
+                                           0x02, 0x03, 0x03, 0x03, 0x04, 0x04,
+                                           0x04, 0x05, 0x05, 0x05, 0x06, 0x06,
+                                           0x06, 0x07, 0x07, 0x07, 0x08, 0x08,
+                                           0x08, 0x09, 0x09, 0x09, 0x0a, 0x0a,
+                                           0x0a, 0x0b, 0x0b, 0x0b, 0x0b, 0x0c,
+                                           0x0c, 0x0c, 0x0d, 0x0d, 0x0d, 0x0e,
+                                           0x0e, 0x0e, 0x0f };
+#endif
+
+#if IS_USED(MODULE_AT86RF233)
+const int16_t _233_tx_pow_to_dbm[16] = { 4, 4, 3, 3, 2, 2, 1,
+                                         0, -1, -2, -3, -4, -6, -8, -12, -17 };
+const uint8_t _233_dbm_to_tx_pow[22] = { 0x0f, 0x0f, 0x0f, 0x0e, 0x0e, 0x0e,
+                                         0x0e, 0x0d, 0x0d, 0x0d, 0x0c, 0x0c,
+                                         0x0b, 0x0b, 0x0a, 0x09, 0x08, 0x07,
+                                         0x06, 0x05, 0x03, 0x00 };
+const int16_t _233_rx_sens_to_dbm[16] = { -101, -94, -91, -88, -85, -82, -79,
+                                          -76, -73, -70, -67, -64, -61, -58, -55, -52 };
+const uint8_t _233_dbm_to_rx_sens[50] = { 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+                                          0x00, 0x01, 0x01, 0x01, 0x02, 0x02,
+                                          0x02, 0x03, 0x03, 0x03, 0x04, 0x04,
+                                          0x04, 0x05, 0x05, 0x05, 0x06, 0x06,
+                                          0x06, 0x07, 0x07, 0x07, 0x08, 0x08,
+                                          0x08, 0x09, 0x09, 0x09, 0x0a, 0x0a,
+                                          0x0a, 0x0b, 0x0b, 0x0b, 0x0c, 0x0c,
+                                          0x0c, 0x0d, 0x0d, 0x0d, 0x0e, 0x0e,
+                                          0x0e, 0x0f };
+#endif
+
+#if IS_USED(MODULE_AT86RF231) || \
+    IS_USED(MODULE_AT86RF232) || \
+    IS_USED(MODULE_AT86RFA1)  || \
+    IS_USED(MODULE_AT86RFR2)
+const int16_t _231_232_a1_r2_tx_pow_to_dbm[16] = { 3, 3, 2, 2, 1, 1, 0,
```

Comment:
Since now all these variables can coexist, we can remove the IS_USED macros. They won't get to the binary file if they are not referenced by someone